docs/compression_algorithms/post_training/Quantization.md

```diff
@@ -0,0 +1,68 @@
+
+
+## Post-Training Quantization
+
+Post-Training Quantization is the quantization algorithm which doesn't demand retraining of a quantized model. 
+It utilizes a small subset of the initial dataset to calibrate quantization constants.
+
+NNCF provides an advanced Post-Training Quantization algorithm, which consists of following techniques:
+
+1) MinMaxQuantization - Analyzes model and inserts extra quantization layers
+are calibrated using the small subset. 
+2) FastBiasCorrection or BiasCorrection - Reduces the bias errors between the quantized layers and the corresponding original layers.
+
+
+### Usage
+
+To start the algorithm the user should provide:
+* Original model.
+* Validation part of the dataset.
+* [Data transformation function](#data-transfomation-function) from original dataset format to the NNCF format.
+
+
+The basic workflow steps:
+1) Create the [data transformation function](#data-transfomation-function).
```

Comment:
Can you add the small code snippets for each step. Something like that should be ok:
```
# Step 1: Initialize transformation function
def transform_fn(data_item):
    images, _ = data_item
    return images
# Step 2: Initialize NNCF Dataset
calibration_dataset = nncf.Dataset(val_dataset, transform_fn)
# Step 3: Run the quantization pipeline
quantized_model = nncf.quantize(model, calibration_dataset)
```

Reply:
Updated